docs/install.rst

```diff
@@ -290,21 +290,20 @@ There's one thing you **must** observe:
 - If your Python is 32 bit you must use the 32 bit versions of those libraries.
 - If your Python is 64 bit you must use the 64 bit versions of those libraries.
 
-If you mismatch the bitness, the warning about kittens dying applies.
+If you install incompatible software architectures, the warning about kittens dying applies.
```

Comment:
Not sure whether *software architecture* is more common than *bitness*

Reply:
architecture is more common in my opinion. OS settings and other books use it.